Spawn-App-iOS-SwiftUI/Views/SubComponents/TagsSubViews/TagsListView.swift

```diff
@@ -15,18 +15,85 @@ struct TagsListView: View {
     }
     
     var body: some View {
-        ScrollView {
-            VStack(alignment: .leading, spacing: 20) {
-                // Title and Time Information
-                ForEach(viewModel.friendTags) { friendTag in
-                    Text(friendTag.displayName)
+        VStack{
+            //search bar
+            searchView
+            
+            ScrollView {
+                //list of tags
+                VStack(alignment: .leading, spacing: 20) {
+                    // Title and Time Information
+                    ForEach(viewModel.friendTags) { friendTag in
+                        HStack {
+                            Text(friendTag.displayName)
+                                .foregroundColor(.white)
+                                .padding()
+                            Spacer()
+                            Image(systemName: "chevron.down")
+                                .foregroundColor(.white)
+                                .padding()
+                        }
+                        .background(friendTag.color)
+                        .cornerRadius(10)
+                        .padding(.horizontal)
+                    }
                 }
+                .padding(20)
```

Comment:
I think you want to remove these three lines to achieve this look:

<img width="174" alt="image" src="https://github.com/user-attachments/assets/fac23c27-f18b-484c-81e6-021920bedc8d">


```suggestion
                .padding(20)
                .background(universalAccentColor)
                .cornerRadius(universalRectangleCornerRadius)
```